stories/HeaderBar.stories.js

```diff
@@ -14,7 +14,7 @@ const customData = {
             keyUiLocale: 'en',
         },
     },
-    '../../dhis-web-commons/menu/getModules.action': {
+    '../dhis-web-commons/menu/getModules.action': {
```

Comment:
We are in a bit of a pickle here...

If we pass in:

```js-jsx
<DataProvider api=''/>
```

Then `../dhis-web-commons/menu/getModules.action` works.

But if we pass in a version to the `dataProvider`:

```js-jsx
<DataProvider api='31'/>
```

We also need to pass in two-steps up: `../../dhis-web-commons/menu/getModules.action`

Of course we are using `''` in feature-toggled apps and `v31` in back-ported apps...

Reply:
Hmm.. imo this should be handled by the data provider logic (prepending `../` to the resources for versioned apis) until we support v32, 33, 34 + dev. Otherwise we'll end up with too many places where we'll have to address this issue with work-around code

Reply:
So this should just be `dhis-web-commons/menu/getModules.action`?

Reply:
This issue will get rid of the hack: https://jira.dhis2.org/browse/DHIS2-7154

Reply:
no, it should stay `../dhis-web-commons/menu/getModules.action`
But when there's a version, the data provider should make it `../../dhis-web-commons/menu/getModules.action` internally

Reply:
> This issue will get rid of the hack: https://jira.dhis2.org/browse/DHIS2-7154

nice!

Reply:
@Mohammer5 would you care to take a look at this PR: https://github.com/dhis2/app-runtime/pull/19 ?

It adds a way to target an action end-point without the `..`-hacks so should work for all cases.

Reply:
Done and commented, very good & necessary patch

Reply:
@Mohammer5 this is available in 1.3.0 of app-runtime.